server/container/jetty/src/main/java/org/apache/james/http/jetty/Configuration.java

```diff
@@ -135,6 +135,13 @@ public Builder port(int port) {
             return this;
         }
 
+        public Builder port(Optional<Integer> port) {
```

Comment:
Of course you can duplicate the method, not the content of the method (:

What about : 
```
Preconditions.checkNotNull(port);
port.ifPresent(p -> port(p));
```